common/src/main/scala/org/apache/celeborn/common/rpc/netty/Inbox.scala

```diff
@@ -218,7 +218,21 @@ private[celeborn] class Inbox(
   /**
    * Calls action closure, and calls the endpoint's onError function in the case of exceptions.
    */
-  private def safelyCall(endpoint: RpcEndpoint)(action: => Unit): Unit = {
+  private def safelyCall(
+      endpoint: RpcEndpoint,
+      endpointRef: NettyRpcEndpointRef)(action: => Unit): Unit = {
```

Comment:
Here is no need to pass NettyRpcEndpointRef here. I think pass NettyRpcEndpointRef.name should could be OK.